cwf/gateway/configs/pipelined.yml

```diff
@@ -147,6 +147,9 @@ clean_restart: true
 
 redis_enabled: false
 
+# Logs grpc payload content
+print_grpc_payload: false
```

Comment:
minor nit: can you rename this to grpc_debug_enabled, sounds more generic than print_grpc_payload

Reply:
Just saw that in sessiond and other places they use MAGMA_PRINT_GRPC_PAYLOAD, maybe we can keep this consistent if we use the same variable.